src/html/searchpreview.html

```diff
@@ -0,0 +1,542 @@
+
+                <p class="search-result-description"></p>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="small-2 columns small-3"></div>
+            <div id="result3" tabindex="0" class="small-10 columns">
+                <h3 class="search-result-link"><a href="" tabindex="-1"></a></h3>
+
+                <p class="search-result-url"></p>
+
+                <p class="search-result-description"></p>
+            </div>
+        </div>
+
+
+        <div class="row">
+            <div class="small-2 columns"></div>
+            <div class="small-10 columns">
+                <ul class="pagination">
+                    <li class="arrow unavailable"><a id="pagination-start" href="" tabindex="-1">«</a></li>
+                    <li class="current"><a id="pagination-1" tabindex="-1" href="">1</a></li>
+                    <li><a href="" id="pagination-2" tabindex="0">2</a></li>
+                    <li><a href="" id="pagination-3" tabindex="0">3</a></li>
+                    <li><a href="" id="pagination-4" tabindex="0">4</a></li>
+                    <li class="unavailable"><a tabindex="-1" href="">…</a></li>
+                    <li><a href="" id="pagination-12" tabindex="0">12</a></li>
+                    <li><a href="" id="pagination-13" tabindex="0">13</a></li>
+                    <li class="arrow"><a id="pagination-end" tabindex="0" href="">»</a></li>
+                </ul>
+            </div>
+        </div>
+
+    </div>
+
+
+    <aside class="small-3 columns hide-for-small">
+        <p id="also_try_label" class="also-try-label" tabindex="0"></p>
+        <ul id="also_try_list" class="also-try-list">
+            <li><a id="also_try_1" tabindex="0"></a></li>
+            <li><a id="also_try_2" tabindex="0"></a></li>
+            <li><a id="also_try_3" tabindex="0"></a></li>
+            <li><a id="also_try_4" tabindex="0"></a></li>
+        </ul>
+    </aside>
+</div>
+
+<footer class="row">
+    <div class="small-12 columns">
+        <hr/>
+        <div class="row">
+            <div class="small-5 columns">
+                <p id="copyright" tabindex="0"></p>
+            </div>
+            <div class="small-7 columns">
+                <ul id="footer_links" class="inline-list right">
+                    <li><a id="footer_link_1" tabindex="0"></a></li>
+                    <li><a id="footer_link_2" tabindex="0"></a></li>
+                    <li><a id="footer_link_3" tabindex="0"></a></li>
+                    <li><a id="footer_link_4" tabindex="0"></a></li>
+                    <li><a id="footer_link_5" tabindex="0"></a></li>
+                </ul>
+            </div>
+        </div>
+    </div>
+</footer>
+<script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/foundation/5.5.3/js/foundation.min.js"></script>
+<script type="text/javascript">
+
+
+    $(document).ready(function () {
```

Comment:
See feedback above on inline JS and CSS, and hard-coded ids, markup, dimensions in JS code